api/controllers/game/create.js

```diff
@@ -1,14 +1,16 @@
 var gameAPI = sails.hooks['customgamehook'];
 
 module.exports = function (req, res) {
+  const { gameName, ranked } = req.body;
```

Comment:
should this have a logical default? 
```suggestion
  const { gameName, ranked = false } = req.body;
```

Reply:
Yeah I think that's reasonable. might let us skip updating some places where I didn't want to include ranked anyway